docs/tutorials/creating-javascript-app-using-alfresco-js-api.md

```diff
@@ -13,9 +13,9 @@ In this tutorial you are going to learn how to develop on top of Alfresco Conten
 
 The only prerequisite of this tutorial is the availability of an instance of Alfresco Content Services in a [Docker](https://www.docker.com/) container. Docker is not the only option you might have for the deployment, but it is the faster we found to focus on the development instead of the environment setup.
 
-If you don't have an instance of Alfresco Content Services up and running check the [preparation of the development environment](./preparing-the-development-environment.html).
+If you don't have an instance of Alfresco Content Services up and running check the [preparation of the development environment](./preparing-environment.html).
```

Comment:
If you use "./preparing-environment.md" as the URL then it will find the file in both Github and the Component Catalog (.md links get translated to .html by the preprocessor).